### PR TITLE
gnu-complexity 1.2 (new formula)

### PR DESCRIPTION
**Pull request by bikeNomad**
I've created a formula for GNU Complexity (which measures C source code complexity).

### CODE REVIEW

File: Library/Formula/gnu-complexity.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"complexity", "--version"
```
Comment (bfontaine) [inline Library/Formula/gnu-complexity.rb]: Can’t we use a test that does something more substantial than e.g. `--help`/`--version`?

File: Library/Formula/gnu-complexity.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class GnuComplexity < Formula
+  desc "Measures complexity of C source"
+  homepage "https://www.gnu.org/software/complexity"
+  url "http://ftpmirror.gnu.org/complexity/complexity-1.2.tar.xz"
```
Comment (DomT4) [inline Library/Formula/gnu-complexity.rb]: Please add `https://ftp.gnu.org/gnu/complexity/complexity-1.2.tar.xz` as a `mirror` line as well, Thanks.

File: Library/Formula/gnu-complexity.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 "22c275e79078bf438dd51108256bb9b33d111330c3f979d7bf1cb0d4aa055e24"
+
+  depends_on "autogen" => :build
+
+  def install
+    system "./configure", "--disable-debug",
```
Comment (DomT4) [inline Library/Formula/gnu-complexity.rb]: `configure: WARNING: unrecognized options: --disable-debug`

File: Library/Formula/gnu-complexity.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/gnu-complexity.rb]: Should be able to write up some very slightly complicated `c` code in the `testpath` and then have it "scored".

**Comment by bfontaine** [comment]
Ping?

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed and audit passing, if you'd like to. Thanks for the submission to Homebrew!

**Pull request closed without merging**